bw2calc/lca.py

```diff
@@ -517,39 +517,47 @@ def redo_lcia(self, demand=None):
         #     extra={"demand": wrap_functional_unit(demand or self.demand)},
         # )
 
-    # def to_dataframe(self, cutoff=200):
-    #     """Return all nonzero elements of characterized inventory as Pandas dataframe"""
-    #     assert mapping, "This method doesn't work with independent LCAs"
-    #     assert (
-    #         pandas
-    #     ), "This method requires the `pandas` (http://pandas.pydata.org/) library"
-    #     assert hasattr(
-    #         self, "characterized_inventory"
-    #     ), "Must do LCIA calculation first"
-
-    #     from bw2data import get_activity
-
-    #     coo = self.characterized_inventory.tocoo()
-    #     stacked = np.vstack([np.abs(coo.data), coo.row, coo.col, coo.data])
-    #     stacked.sort()
-    #     rev_activity, _, rev_bio = self.reverse_dict()
-    #     length = stacked.shape[1]
-
-    #     data = []
-    #     for x in range(min(cutoff, length)):
-    #         if stacked[3, length - x - 1] == 0.0:
-    #             continue
-    #         activity = get_activity(rev_activity[stacked[2, length - x - 1]])
-    #         flow = get_activity(rev_bio[stacked[1, length - x - 1]])
-    #         data.append(
-    #             (
-    #                 activity["name"],
-    #                 flow["name"],
-    #                 activity.get("location"),
-    #                 stacked[3, length - x - 1],
-    #             )
-    #         )
-    #     return pandas.DataFrame(data, columns=["Activity", "Flow", "Region", "Amount"])
+    def to_dataframe(self, cutoff=200):
+        """Return all nonzero elements of characterized inventory as Pandas dataframe"""
+        # assert mapping, "This method doesn't work with independent LCAs"
+        assert (
+            pandas
```

Comment:
pandas is now part of the install_requires of the package, we could leave this out.